### PR TITLE
WIP: adding shap

### PR DESCRIPTION
**Pull request by mrandrewandrade**

### CODE REVIEW

**Comment by mrandrewandrade** [comment]
Should be g2g here!  LMK if a change is required.

File: recipes/shap/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,55 @@
...
+  summary: 'A unified approach to explain the output of any machine learning model.'
+  description: |
+    SHAP (SHapley Additive exPlanations) is a unified approach to explain the output of any machine learning model. SHAP connects game theory with local explanations, uniting several previous methods and representing the only possible consistent and locally accurate additive feature attribution method based on expectations.
+  doc_url: https://shap.readthedocs.io/en/latest/
+  dev_url: https://github.com/slundberg/shap
+
+extra:
+  recipe-maintainers:
+    - mrandrewandrade 
+    - slundberg
```
Comment (synapticarbors) [inline recipes/shap/meta.yaml]: Can you confirm that the co-maintainers have agreed to be listed here? If not, please ping them to get a response.

**Comment by mrandrewandrade** [comment]
@synapticarbors Thanks for the feedback, I made the changes.  Please let me know if we are missing anything else!

**Comment by synapticarbors** [comment]
@mrandrewandrade -- Thanks for your contribution to conda-forge. A feedstock for this recipe should be generated soon and you'll be receiving several automated messages about it. When the feedstock has been rendered, please take a look at the README in that repo for information about updating and maintaining the recipe. One of the key points is all updates should be done by forking that repo and following a standard pull request workflow. If you haven't looked at it already the docs are quite helpful:

https://conda-forge.org/docs/

If you run into any difficulties please let us know.

**Pull request merged**